e2e/src/config/tests-config/accounts/account-manager.ts

```diff
@@ -48,72 +53,61 @@ abstract class AccountManager implements IAccountManager {
     return { account: whaleAccount, accountWallet: whaleTxManager };
   }
 
```

Comment:
Just to confirm, each of the parellelly running e2e tests will have its own `JEST_WORKER_ID`, right?

Reply:
yes each tests suite will have its own JEST_WORKER_ID. So if 5 tests suites are running in parallel, JEST_WORKER_IDs will be 1, 2, 3, 4, 5. So each test suite will use a different whale account